aiosmtpd/controller.py

```diff
@@ -57,7 +64,9 @@ def start(self):
         self.thread.daemon = True
         self.thread.start()
         # Wait a while until the server is responding.
-        ready_event.wait()
+        ready_event.wait(self.thread_timeout)
+        if self.thread_exception:
```

Comment:
I missed this the last time, but since this is either going to be None or some exception instance, it won't be any other false-y value.  I prefer an explicit test against None, e.g.  `if self.thread_exception is not None:`